LibTest/js_interop_unsafe/JSObjectUnsafeUtilExtension/delete_A01_t05.dart

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion JSBoolean delete( JSAny property )
+/// Deletes the property with key `property` from this [JSObject].
+///
+/// @description Check that this function returns `true`.
```

Comment:
I think we should receive `false` as the return value if, for example, we try to remove the `length` from an array.

Reply:
Sounds good! @sgrekhov, are you planning to add that extra test now?

Reply:
I missed that. Yes, I’ll add a test.